deploy/config/microsoft/cloudconfig.yml

```diff
@@ -62,7 +62,7 @@ cloud:
         # dnsServiceIp: defaults to 10.0.0.10
 
         systemNodePool:
-          name: system
+          name: systemnp
```

Comment:
Is this to make sure we don't have conflicts when upgrading env?

Reply:
Yes. You can't change the sky on a node pool. We handle this by creating a new node pool, then deleting the existing one.